graphql/queries.graphql

```diff
@@ -71,3 +71,55 @@ query GetClientOrders($email: String!) {
 		id
 	}
 }
+
+query GetFirstProducts($size:Int) {
+  productsConnection(first:$size) {
+    edges{
+      cursor
+      node{
+        id
+		name
+		price
+		slug
+		images {
+			url
+		}
+		description
+		reviews {
+			rating
+		}
+      }
+    }
+    pageInfo{
+      endCursor
+      hasNextPage
+    }
+  }
+}
+
+
+query GetMoreProducts($cursor:String!,$size:Int!) {
```

Comment:
GetMoreProduct and GetFirstProducts are same.
You can create GetProductsCursor one hook and provide $size:int and $cursor:String without !. If the cursor will not provided then we get the first products.